cz.zeropage.Formiko.json

```diff
@@ -0,0 +1,300 @@
+{
+    "app-id": "org.flatpak.Formiko",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "formiko",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=session-bus",
+        "--filesystem=home",
```

Comment:
If you use `GtkFileChooserNative`, files will be opened via portal and you'll be able to drop these holes in sandbox.

Reply:
I'm not using GtkFileChooserNative at all, but i try to fix it, so this could be not defined.